### PR TITLE
Fix broken graphql-config links in README

### PR DESCRIPTION
**Pull request by pleunv**
Noticed that the old links are now returning a 404, updated them to the correct pages. Scanned the rest of the repo for other instances of graphql-config hyperlinks but these appear to be the only ones.

### CODE REVIEW

**Review by thomasheyenbrock — COMMENTED** [review]
Awesome, thanks for this! 👏 

Before we can merge your contribution to this repo you have to sign the CLA, just click the link in the CLA bot comment and that should walk you through everything 🙂

**Comment by acao** [comment]
Thank you, the path to this page changes every year, I wish they would just use 301 redirects 😅

File: packages/graphql-language-service-server/README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ When developing vscode extensions, just the above is enough to get started for y
 
 You _must_ provide a graphql config file
 
-Check out [graphql-config](https://graphql-config.com/introduction) to learn the many ways you can define your graphql config
+Check out [graphql-config](https://www.graphql-config.com/docs) to learn the many ways you can define your graphql config
```
Comment (B2o5T) [inline packages/graphql-language-service-server/README.md:75]:
```suggestion
Check out [graphql-config](https://the-guild.dev/graphql/config/docs) to learn the many ways you can define your graphql config
```

File: packages/graphql-language-service-server/README.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ module.exports = { schema: 'https://localhost:8000' };
 
 #### custom `startServer`
 
-use graphql config [`loadConfig`](https://graphql-config.com/load-config) for further customization:
+use graphql config [`loadConfig`](https://www.graphql-config.com/docs/library/load-config) for further customization:
```
Comment (B2o5T) [inline packages/graphql-language-service-server/README.md:106]:
```suggestion
use graphql config [`loadConfig`](https://the-guild.dev/graphql/config/docs/library/load-config) for further customization:
```

**Comment by B2o5T** [comment]
@acao we added redirects for these links so I just close this PR

**Pull request closed without merging**